docs/using.md

```diff
@@ -1,166 +1,428 @@
-to add actors and workloads. For details on the development process, see [Developing Genny](./developing.md).
+# Table of Contents
 
-## Running Genny Workloads
+1.  [Introduction](#orgf1f92f6)
+2.  [Getting Started and Building](#org35e6dff)
+3.  [Core Concepts](#org1140a6b)
+    1.  [What is load generation?](#orgdcd1898)
+    2.  [What is the system under test?](#orgc7904ae)
+    3.  [What is a workload?](#org3610c67)
+        1.  [How are workloads configured?](#orgdecc7ae)
+        2.  [What is an actor?](#org51d4d33)
+        3.  [What is a phase?](#orgb655d69)
+        4.  [How do I run a workload?](#org32b8ad3)
+    4.  [Outputs](#orgec88ad4)
+    5.  [Workload Development](#org0e7c476)
+4.  [Further Concepts](#org61c719c)
+    1.  [Common Actors](#org78b250a)
+    2.  [AutoRun](#org2b04b49)
+        1.  [What is AutoRun?](#orgd0067d1)
+        2.  [Configuring AutoRun](#orgbfb0d8e)
+    3.  [Generators](#orgd89f221)
+    4.  [Preprocessor](#org2078b23)
+        1.  [LoadConfig](#orga6d35c7)
+        2.  [ActorTemplate](#orga45b1d8)
+        3.  [OnlyActiveInPhases](#orgf9c328f)
+        4.  [Defaults and Overrides](#org22b7a0f)
+    5.  [Connecting to the Server](#orgd6b0450)
+        1.  [Connection Strings and Pools](#orgd2659db)
+        2.  [Multiple Connection Strings](#orga591018)
+        3.  [Default](#org65830c2)
+    6.  [Creating an Actor](#org7e6c6bd)
+5.  [Pitfalls](#org3aaae9e)
+    1.  [pipe creation failed (24): Too many open files](#orga7ab911)
+    2.  [Actor integration tests fail locally](#orgb084b49)
+    3.  [The Loader agent requires thread count set on both actor and phase level](#org97681a9)
 
-Start a mongod process. Use locally-built mongod if available. Otherwise, first install mongodb:
 
-```sh
-brew install mongodb
-mongod --dbpath=/tmp
-```
+<a id="orgf1f92f6"></a>
 
-Then build Genny (see [above](#build-and-install) for details):
+# Introduction
 
-And then run a workload:
+Hello! These are the docs for Genny specifically. For an overall view of MongoDB's performance testing infrastructure, please look at the [Performance Tooling Docs](https://github.com/10gen/performance-tooling-docs).
 
-```sh
-./run-genny workload ./src/workloads/scale/InsertRemove.yml
-```
-If you see errors like the following, try reducing the number of threads and duration in ./src/workloads/scale/InsertRemove.yml
-```E0218 17:46:13.640106000 123145604628480 wakeup_fd_pipe.cc:40]         pipe creation failed (24): Too many open files```
+If you have any questions, please reach out to the TIPS team in our dedicated slack channel: #performance-tooling-users. If you feel like these docs can be improved in any way, feel free to open a PR and assign someone from TIPS. No ticket necessary. This document is intended to be readable straight-through, in addition to serving as a reference on an as-needed basis. If there are any difficulties in flow or discoverability, please let us know.
```

Comment:
wrt discoverability - I tried searching "genny" in https://wiki.corp.mongodb.com/ and a link to this doc did not show up. Maybe we can add a link somewhere in our wiki to ensure that this doc comes up as one of the results when searching for genny.

Reply:
Good idea! Will do, though outside the context of this PR (and this should probably be merged first)